### DISCUSSION TITLE
Fine-tuning Stable Diffusion -  unconditional guidance scale

### DISCUSSION THREAD

**Discussion by Arwa-97** [keras-team/keras-cv#2261] [Q&A] [answered]
Greetings,
in the Fine-tuning Stable Diffusion what is the max and min value for unconditional guidance scale?

**Comment by divyashreepathihalli** [comment, marked as answer]
No specific limit. Larger values result in more closely adhering to the prompt, but will make the image noisier. Defaults to 7.5.
